flight/util/json_parsing.py

```diff
@@ -0,0 +1,29 @@
+import json
```

Comment:
please write a file docstring, a one-liner would do saying what the file is for/what it does